src/Platform/Microsoft.Testing.Platform/Helpers/System/SystemFileSystem.cs

```diff
@@ -17,14 +17,5 @@ internal sealed class SystemFileSystem : IFileSystem
 
     public string ReadAllText(string path) => File.ReadAllText(path);
 
-#if NETCOREAPP
     public Task<string> ReadAllTextAsync(string path) => File.ReadAllTextAsync(path);
```

Comment:
I think the polyfill doesn't use `FileOptions.SequentialScan`, so this potentially introduces performance difference.

Reply:
i did look at that. it is recommended for large files. and is slower on non windows for smaller files. 


so thoughts? should polyfill move to sequential?

Reply:
I think we might have been just replicating what `File.ReadAllTextAsync` API does in .NET Core. https://github.com/dotnet/runtime/blob/c1523bbb379371a75ff28a31748186700396ec1a/src/libraries/System.Private.CoreLib/src/System/IO/File.cs#L1077-L1082

So probably it wasn't measured and was used like that from the beginning.

So far, it seems to only be used by VSTestBridge to read runsettings.

If it can really be slower, then maybe we shouldn't even be using `File.ReadAllTextAsync` at all and use the lower-level StreamReader directly.

Reply:
I started a mirror of beta.12 which will roughly complete in half an our. Then you can update the package version in the PR.